src/index.ts

```diff
@@ -1,10 +1,6 @@
 export * from './window/WindowPostMessageStream';
```

Comment:
Nit: What are your thoughts on converting these to named exports rather than using barrel exports?

Reply:
Sounds good to me. Any idea how we can keep the coverage up when doing so? Jest is complaining that the exports aren't covered in this case, and I don't think we should duplicate all imports in the test 🤔 

<img width="795" alt="image" src="https://github.com/user-attachments/assets/526d3dcb-9bf6-4105-810d-f79ba3f53c2a" />

Reply:
Oh, really? Maybe that explains why we ignore this file in the coverage report in some packages, even if we have an `index.test.ts`. 

Maybe if we switch away from barrel exports we don't need an `index.test.ts`, since all the exports will be clearly named. I guess that could be done in another PR, though.

So maybe keeping the barrel exports is okay for now.

Reply:
Yeah, I guess the test isn't necessary if everything is explicitly exported. We can revisit this in a later PR.

I just added an export and test for `isValidStreamMessage` since it's used by Snaps, but currently imported through `@metamask/post-message-stream/dist/utils`.